modules/encounter.py

```diff
@@ -212,6 +229,12 @@ def handle_encounter(
                 save_pk3(pokemon)
             is_of_interest = True
 
+        case EncounterValue.AvoidedPokemon:
+            console.print(f"Attempting to avoid {pokemon.species.name} because it is on avoided list")
+            EncounterValue.avoid = True
+            alert = None
+            is_of_interest = False
+
```

Comment:
```suggestion
        case EncounterValue.AvoidedPokemon:
            console.print(f"Attempting to avoid {pokemon.species.name} because it is on avoided list")
            alert = None
            is_of_interest = False
            
```